### PR TITLE
feat: expose prometheus metrics via jmx exporter

### PR DESCRIPTION
**Pull request by iromli**
### Prepare

- [x] Read [PR guidelines](https://github.com/JanssenProject/jans/blob/main/docs/CONTRIBUTING.md#prs)
- [x] Read [license information](https://github.com/JanssenProject/jans/blob/main/LICENSE)

-------------------

#### Target issue

GH issue: #682 

#### Implementation Details

Prometheus metrics can be exposed via JMX exporter by setting the newly added `CN_PROMETHEUS_PORT` env, i.e. `CN_PROMETHEUS_PORT=9093`. The metrics can be scraped by  Prometheus or accessed manually by making request to `http://$SERVICE:9093/metrics` URL.

-------------------
### Test and Document the changes
- [x] Relevant documentation has been updated if any (i.e. user guides, installation and configuration guides, technical design docs etc); see `docker-jans-*/README.md` for details

### CODE REVIEW

File: docker-jans-auth-server/Dockerfile [hunk 1]
```diff
@@ -135,6 +135,15 @@ RUN python3 -m ensurepip \
     && pip3 install --no-cache-dir --default-timeout=300 -r /app/requirements.txt \
     && pip3 uninstall -y pip wheel
 
+# ==========
+# Prometheus
+# ==========
+
+COPY conf/prometheus-config.yaml /opt/prometheus/
+RUN mkdir -p /opt/prometheus \
+    && wget -q https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/0.17.0/jmx_prometheus_javaagent-0.17.0.jar -O /opt/prometheus/jmx_prometheus_javaagent.jar \
```
Comment (moabu) [inline docker-jans-auth-server/Dockerfile]: Lets follow the same convention and specify an env and date for the version so it can get easily updated

**Review by moabu — CHANGES_REQUESTED** [review]
- [ ] I think it might be suitable to add this across our other services @iromli . Let me know your thoughts.
- [ ] Add the required work for the helm charts @moabu

File: docker-jans-auth-server/Dockerfile (hunk 1 above)
Comment (iromli) [reply docker-jans-auth-server/Dockerfile]: we can add env for `jmx_prometheus_javaagent` version, but i don't think date will be necessary as we will fetch stable release

**Comment by iromli** [comment]
> * [ ]  I think it might be suitable to add this across our other services @iromli . Let me know your thoughts.

@moabu Yes, we can add `jmx_prometheus_javaagent` to all jetty-based services (`auth-server`, `config-api`, `scim`, `fido2`, `client-api`)

**Review by moabu — APPROVED** [review]

**Comment by mecsys** [comment]
Very nice feature!

**Pull request merged**